debian/precise/foreman/foreman-debug.dirs

```diff
@@ -0,0 +1 @@
+usr/share/foreman/script/foreman-debug.d/
```

Comment:
It's an extension point, example plugin: https://github.com/Katello/katello/blob/master/deploy/script/katello-debug.sh